.github/workflows/e2e.yml

```diff
@@ -40,8 +40,15 @@ jobs:
       -
         name: Update daemon.json
         run: |
-          sudo jq '.experimental = true' < /etc/docker/daemon.json > /tmp/docker.json
-          sudo mv /tmp/docker.json /etc/docker/daemon.json
+          if [ ! -f /etc/docker/daemon.json ]; then
+            # ubuntu 24.04 runners no longer have a default daemon.json present
```

Comment:
Yeah, that was my thinking; if they decide to set some options, we wouldn't immediately break.

But the `cat /etc/docker/daemon.json` below should show us what's in there, in case their changes would cause a change in behavior and we wanted to see "what changed?"

Reply:
Yeah, i noticed that..I guess that can work, no hard feelings here anyway :)

Reply:
Thanks! Yeah, I guess it's one of those cases where there's pros/cons to either approach.


Let me bring this one in; thanks for reviewing!